Dockerfile

```diff
@@ -1,19 +1,41 @@
-FROM node:14-alpine
+# application builder
+FROM node:16-alpine AS builder
 
-# directory for the app in the container
 WORKDIR /usr/app
 
-# copies all the app's files from host into the container folder which
-# might include the node_modules dir if npm install executed in the host
+# copy package.json and package-lock.json files
+COPY package*.json .
+
+# install dependencies including local development tools
+RUN npm install
+
+# copy the rest of the content
 COPY . /usr/app
 
-# removes any existing node_modules folder
-# this prevents the host's node_modules from being used in the container
-# which could cause issues with native binaries such as node_sass.
-RUN rm -rf /usr/app/node_modules/
+# build the application and generate a distribution package
+RUN npm run build
+
+# application package
+FROM node:16-alpine AS app
 
+WORKDIR /usr/app
+
+ENV NODE_ENV=production
 ENV CYPRESS_INSTALL_BINARY=0
 
-RUN npm install -g npm@7.21.0 && \
-    npm install && \
-    npm run i18n:get-translations
```

Comment:
Please restore this `npm run i18n:get-translations` script. This script downloads all of the translation json for the different languages supported by Openverse.

Reply:
@zackkrida 
`i18n:get-translations` is already present in the `build` script, where I feel is the right place to locate this script.

Since I'm building a distribution package during the `builder` stage ([line 16 in the Dockerfile](https://github.com/WordPress/openverse-frontend/blob/ci/pipelines/Dockerfile#L16)) the files are transferred as part of the source.

Reply:
You're right!